ui/bin/roll20.js

```diff
@@ -1,10 +1,3 @@
-(function() {
-  /**
-   * This is a test function that has no effect.
-   * Just testing the mergeScripts.sh
-   * This is also an example of an IIFE, which allows each script to be self-contained to reduce risks of collisions between scripts.
-   */
-})();
 // eslint-disable-next-line no-unused-vars
 const prevEPRoll = function() {
```

Comment:
It worked! Unsure why it rearranged things, but I am guessing we can adjust the `find` script to be case insensitive order.